omrmakefiles/rules.aix.mk

```diff
@@ -156,6 +156,14 @@ endif
 
 ## Debug Information
 ifeq (1,$(OMR_DEBUG))
-  GLOBAL_CXXFLAGS+=-g
-  GLOBAL_CFLAGS+=-g
+  GLOBAL_CXXFLAGS+=-g -qsuppress=1540-0825
+  GLOBAL_CFLAGS+=-g -qsuppress=1540-0825
+  GLOBAL_CPPFLAGS+=-qsuppress=1540-0825
+endif
+
+#DDR compile
+ifeq (yes,$(ENABLE_DDR))
+  GLOBAL_CXXFLAGS+=-qsuppress=1540-0825
```

Comment:
Can you provide an example of a file which generates the warning and the reason for the warning?  It seems very heavy handed to add this option to the global flags when DDR or DEBUG is enabled.

Reply:
ddrgen's getmacro script annotates header files with macros it finds with lines starting with `@`. During the preprocessor stage, xlC generates invalid character warnings for each line like that it finds, while GCC did not.

Reply:
Does it annotate the files for normal compilation or just for compilation during ddrgen processing?  If it is the later then these options should only be added for one of the ddr phases and not everywhere.

Reply:
I think some dependencies are missing in openj9: if they were added, the compiler wouldn't be invoked while the header files are mutated by getmacros.

Reply:
It still feels very heavy handed to set these flags globally if they are not needed.  Also as OpenJ9 moves towards CMake do we need changes in the CMake build system as well?

Reply:
I spoke with @mikezhang1234567890 about reorganizing the makefiles in OpenJ9 so we won't have such warnings.